website/sidebars.js

```diff
@@ -333,6 +333,7 @@ module.exports = {
                         'reference/unleash-context',
                         'reference/change-requests',
                         'reference/maintenance-mode',
+                        'reference/notifications',
```

Comment:
Just realized maintenance mode is also in the wrong place. Would you mind placing these last two around network view, so that it goes:

```markdown
- ...
- login-history
- maintenance-mode
- network-view
- notifications
- ...
```